x-pack/test_serverless/functional/test_suites/security/ftr/cases/helper/api.ts

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
This file is basically the same as `x-pack/test_serverless/functional/test_suites/observability/cases/helper/api.ts` and the helpers there don't look to be specific to serverless. Also it seems we already very similar functionality in the cases service here: https://github.com/elastic/kibana/blob/main/x-pack/test/functional/services/cases/api.ts, so I would suggest to re-use what we have there or, if necessary, to make any additions and adjustments in that service, which is usable across the different test areas we have.

Reply:
fixed in 20516f1